src/components/scenes/FioRequestListScene.js

```diff
@@ -486,28 +485,21 @@ export class FioRequestList extends React.Component<Props, LocalState> {
       })
       headers.push({ title: previousTitle, data: requestsInSection })
     }
-
     return headers
   }
 
   listKeyExtractor(item: FioRequest) {
     return item.fio_request_id.toString()
   }
 
-  renderPending = (itemObj: { item: FioRequest, index: number }) => {
-    const { item: fioRequest, index } = itemObj
-    const isLastOfDate =
-      index + 1 === this.state.fioRequestsPending.length ||
-      (index > 0 && formatDate(new Date(this.state.fioRequestsPending[index + 1].time_stamp)) !== formatDate(new Date(fioRequest.time_stamp)))
-    return <FioRequestRow fioRequest={fioRequest} isLastOfDate={isLastOfDate} onSelect={this.selectPendingRequest} />
+  renderPending = (itemObj: { item: FioRequest }) => {
+    const { item: fioRequest } = itemObj
+    return <FioRequestRow fioRequest={fioRequest} onSelect={this.selectPendingRequest} />
   }
 
-  renderSent = (itemObj: { item: FioRequest, index: number }) => {
-    const { item: fioRequest, index } = itemObj
-    const isLastOfDate =
-      index + 1 === this.state.fioRequestsSent.length ||
-      (index > 0 && formatDate(new Date(this.state.fioRequestsSent[index + 1].time_stamp)) !== formatDate(new Date(fioRequest.time_stamp)))
-    return <FioRequestRow fioRequest={fioRequest} onSelect={this.selectSentRequest} isSent isLastOfDate={isLastOfDate} />
+  renderSent = (itemObj: { item: FioRequest }) => {
+    const { item: fioRequest } = itemObj
+    return <FioRequestRow fioRequest={fioRequest} onSelect={this.selectSentRequest} isSent />
   }
 
   renderHiddenItem = (rowObj: { item: FioRequest }, rowMap: { [string]: SwipeRow }) => {
```

Comment:
HiddenMenuButtons should have the same behavior as current wallet list swipe row